UnityWeld_Editor/AnimatorParameterBindingEditor.cs

```diff
@@ -82,7 +82,8 @@ out viewPropertyType
 
             var viewAdapterTypeNames = GetAdapterTypeNames(
                 type => viewPropertyType == null ||
-                    TypeResolver.FindAdapterAttribute(type).OutputType == viewPropertyType
+                    TypeResolver.FindAdapterAttribute(type).OutputType == viewPropertyType ||
+                    TypeResolver.IsTypeCastableTo(TypeResolver.FindAdapterAttribute(type).OutputType, viewPropertyType, true)
```

Comment:
If we put the `t1 == t2` check inside TypeResolver.IsTypeCastableTo that would a lot of duplication and we wouldn't need to call FindAdapterAttribute twice. 

This applies to all the changes in the editors.